src/containers/PreviewDraftPage/PreviewDraftPage.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Fragment, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Hero, OneColumn } from '@ndla/ui';
+import * as draftApi from '../../modules/draft/draftApi';
+import * as articleApi from '../../modules/article/articleApi';
+import PreviewDraft from '../../components/PreviewDraft/PreviewDraft';
+import { queryResources } from '../../modules/taxonomy';
+import { getContentTypeFromResourceTypes } from '../../util/resourceHelpers';
+import PreviewDraftLanguages from './PreviewDraftLanguages';
+
+const PreviewDraftPage = ({
+  match: {
+    params: { draftId, language },
+  },
+}) => {
+  const [draft, setDraft] = useState(undefined);
+  const [resource, setResource] = useState(undefined);
+
+  const fetchDraft = async () => {
+    const fetchedDraft = await draftApi.fetchDraft(draftId, language);
+    const convertedArticle = await articleApi.getPreviewArticle(
+      fetchedDraft,
+      language,
+    );
+    setDraft(convertedArticle);
+  };
+
+  const fetchResource = async () => {
+    const fetchedResource = await queryResources(draftId, language);
+    setResource(fetchedResource);
+  };
+
+  useEffect(() => {
+    fetchDraft();
+    fetchResource();
+  }, [draftId, language]);
+
+  if (!draft) {
+    return null;
+  }
+
+  const contentType = resource
+    ? getContentTypeFromResourceTypes(resource[0].resourceTypes)
```

Comment:
Denne gir nullpointer for enkelte artikler. resource[0] er undefined.

Reply:
Har lagt inn sjekk på lengde nå, men hvilken artikler gjelder det?